CodeStarterCamp_Week1/main.swift

```diff
@@ -8,5 +8,17 @@
 
 import Foundation
 
-print("Hello, World!")
+func makeIcecream(width: Int, height: Int, shape: String) {
+    for _ in 1...height {
+        print(String(repeating: shape, count: width))
+    }
+}
 
+func makeStick(height: Int,shape: String) {
```

Comment:
사소해보일 수도 있지만, 공백을 적절히 삽입하는 것도 신경써주세요 ☺️
```suggestion
func makeStick(height: Int, shape: String) {
```